src/utils/numbers.js

```diff
@@ -47,9 +46,8 @@ export function toFixed(x) {
 }
 
 export const calculateMaxBorrow = (creditLimit, originationFee) => {
-  const cl = Number(creditLimit.mul(99999).div(100000).toString());
-  const ofe = Number(originationFee.toString()) / 1e18;
-  return BigNumber.from(toFixed(Math.floor(cl / (ofe + 1)).toString()));
+  const borrowFee = creditLimit.mul(originationFee).div(WAD);
+  return creditLimit.sub(borrowFee);
```

Comment:
I think instead of this we should just change:
```js
const cl = Number(creditLimit.mul(99999).div(100000).toString());
```
```js
const cl = Number(creditLimit.mul(9999).div(10000).toString());
```
As this calculation of just subbing the origination fee of the max credit limit gets worse as the credit limit increases.
Consider a million dollar credit line:
```
creditLimit = 10,000,000           = 10,000,000
fee = creditLimit * 0.05           = 500,000
r = creditLimit - fee              = 9,500,000

actual = creditLimit / (1 + 0.05)  = 9,523,809.52381
delta = actual - r                 = 23,809.52381

```